classes/class-p4-campaign-importer.php

```diff
@@ -125,6 +125,18 @@ public function update_campaign_attachements( $post_id, $original_post_id, $post
 					$campaign_postmeta = $post['postmeta'];
 				}
 
+				// Search and replace the special char in post content.
+				$replace_chars = [
+					'u0026' => '\u0026',
```

Comment:
Shouldn't this replace `'u0026'` only when it is not preceded by a `\`?

Reply:
Here the slash is already there, but on save of post content it cleaned up. the patch added here will add additional slash for `\u0026 , \u0022 , \u003c and \u003e` special chars which for now solves the issue.